src/plugin/memory-webview-main.ts

```diff
@@ -49,6 +52,12 @@ enum RefreshEnum {
     on = 1
 }
 
+export interface WebviewMenuContext {
+    messageParticipant: WebviewIdMessageParticipant,
+    visibleColumns: string,
+    showRadixPrefix: boolean,
+}
```

Comment:
Same as `AppVscodeContext` in `vscode-contexts.ts`. Perhaps it should be shared in the common area?